scylla-cdc-printer/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "scylla-cdc-printer"
-version = "0.1.1"
+version = "0.1.2"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-scylla = "0.13.1"
-scylla-cdc = { version = "0.2.0", path = "../scylla-cdc" }
+scylla = "0.15.1"
```

Comment:
:wrench: As we're bumping a major version number of `scylla`, let's bump a major version number of dependant packages, too.
(`0.2.0` instead of `0.1.2`)

Reply:
`scylla-cdc-printer` is a command line app, version bumps of `scylla` don't affect it that way.
For a command line app the closes thing we have to public API is its interface (arguments, flags etc). If those don't change, then I see no reason to bump the major version.
The same is true for `scylla-cdc-replicator`.